include/SH3/system/glprogram.hpp

```diff
@@ -73,6 +73,14 @@ namespace sh3_gl
          */
         void Unbind();
 
+        /**
+         *  Gets the OpenGL generated Program ID from instance of this (shader) class so that
```

Comment:
The first sentence is a short description. I'd write something like "Gets the Program ID.". Then have an empty line, followed by the long description, if any is necessary.